proposals/lip-0050.md

```diff
@@ -171,14 +239,47 @@ registerBLSKeyParamsSchema = {
         "proofOfPossession": {
             "dataType": "bytes",
             "fieldNumber": 2
+        },
+        "generatorKey": {
+            "dataType": "bytes",
+            "fieldNumber": 3
         }
     }
 }
 ```
 
+#### Verification
+
+```python
+def verify(trs: Transaction) -> None:
+    validatorAddress = 20-byte address derived from trs.senderPublicKey
+
+    if validator.getValidatorAccount(validatorAddress) does not exists :
+        raise Exception('Public key does not correspond to a registered validator.')
+        
+    validatorAccount = validator.getValidatorAccount(validatorAddress)
+    if (validatorAccount.generatorKey != INVALID_ED25519_KEY
```

Comment:
The constant `INVALID_ED25519_KEY` is not defined in this LIP. Add it to the constants. Also probably some rationale and explanation would be helpful.

Reply:
Added it to the table. I will keep the explanation minimal here, the reasoning behind the existence of `INVALID_ED25519_KEY` should be in the migration LIP.

Reply:
Please double check the value.

Reply:
The value is correct.